apps/fyle/helpers.py

```
@@ -97,14 +97,14 @@ def upload_iif_to_fyle(file_path: str, workspace_id: int):
         ]
     }
 
-    upload_url = platform.v1beta.admin.files.bulk_generate_file_urls(
+    upload_url = platform.v1.admin.files.bulk_generate_file_urls(
         bulk_generate_file_urls_payload
     )['data'][0]['upload_url']
 
     file_data = open(file_path, 'rb')
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)